addons/web/static/src/core/navigable_popover/components/dropdown.js

```diff
@@ -0,0 +1,201 @@
+/** @odoo-module **/
+
+import { Component, xml, useState, useRef, useEffect } from "@odoo/owl";
+import { ParentClosingMode, useMenuHasParent } from "./nesting";
+import { Menu, MenuItem } from "./menu";
+import { useRefListener } from "@web/core/utils/hooks";
+
+const GROUPS = new Map();
+function useGroups(togglerRef, state) {
+    // The key of a group is the parent element of the toggle,
+    // meaning only dropdowns under the same element will work
+    // together.
```

Comment:
Just a small comment on this specific behavior: it is not written in stone :-)

I mean that maybe we want to change this ?